roles/rudder_agent/tasks/configure.yml

```diff
@@ -5,3 +5,21 @@
     owner: root
     group: root
     mode: '0644'
+
+- name: Set custom Rudder ports for 6 (min) version
+  replace:
```

Comment:
won't this conflict with the file content just above?

Reply:
it doesn't seem to be in conflict, in any case, no problem during my tests.

Reply:
doesn't it make 2 repairs every time it runs? (on to reset the content of the file to the policy server hostname and one to set the port?